inspirehep/__init__.py

```diff
@@ -1,9 +1,30 @@
 # -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015, 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
 
-"""inspirehep."""
+"""INSPIREHEP."""
 
-from __future__ import absolute_import, print_function
+from __future__ import absolute_import, division, print_function
```

Comment:
It's a bit useless, but I try to have these future imports everywhere for Python 3 compatibility. Now, it's not doing anything here, but I prefer to have them so people have less chances of copy pasting the wrong boilerplate.